UIComponets/ViewController/UITextFieldVC.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  UITextField.swift
+//  UIComponets
+//
+//  Created by Rajan Patel on 27/04/23.
+//
+
+import UIKit
+
+class UITextFieldVC: BaseViewController {
+    
+    // MARK: - IBOutlets
+    @IBOutlet private weak var btnSubmit: UIButton!
+    @IBOutlet private weak var txtUserName: UITextField!
+    
+    @IBOutlet private weak var txtPassword: UITextField!
+    @IBOutlet private weak var txtPincode: UITextField!
+    @IBOutlet private weak var txtAddress: UITextField!
+    @IBOutlet private weak var txtEmail: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        txtUserName.delegate = self
+        txtAddress.delegate = self
+        txtPassword.delegate = self
+        txtPincode.delegate = self
+        txtEmail.delegate = self
```

Comment:
avoid adding implementation directly in viewdidload instead create function for same